presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -51,8 +51,33 @@
 public class TestHiveTransactionalTable
         extends HiveProductTest
 {
-    @Test(groups = {STORAGE_FORMATS, HIVE_TRANSACTIONAL}, dataProvider = "partitioningAndBucketingTypeDataProvider", timeOut = 5 * 60 * 1000)
+    private static final int TEST_TIMEOUT = 5 * 60 * 1000;
+
+    @Test(groups = STORAGE_FORMATS, timeOut = TEST_TIMEOUT)
+    public void testAcidSmokeTest()
+    {
+        doTestReadFullAcid(true, BucketingType.BUCKETED_DEFAULT);
+    }
+
+    @Test(groups = HIVE_TRANSACTIONAL, dataProvider = "partitioningAndBucketingTypeDataProvider", timeOut = TEST_TIMEOUT)
```

Comment:
I have doubts if mixing manual test definitions with data provider makes thinks more readable.
I would go with inlining every thing if we need to inline some.

Reply:
i started with the approach you propose. 
However, since test has the timeout definition, it required to create test method
for each of the tested combinations. The code was less readable IMO, so i
switched back to the test provider.

Reply:
Would this be helpful?
https://stackoverflow.com/a/30802395

Reply:
I'd prefer KISS

Reply:
Yeah - that would mean separate methods for me :) 
Anyway LGTM

Reply:
I pushed a version without DP for testReadFullAcid, let me know if it reads better for you.

the DP is retained for the insert-only test still.

Reply:
Thanks. I like it more. It may that I am just old :)